src/components/SamlConfiguration/SamlProviderConfigForm.jsx

```diff
@@ -0,0 +1,360 @@
+                type="text"
+                id="attrFullName"
+                name="attrFullName"
+                defaultValue={config ? config.attrFullName : ''}
+              />
+            </ValidationFormGroup>
+          </div>
+        </div>
+        <div className="row">
+          <div className="col col-4">
+            <ValidationFormGroup
+              for="attrFirstName"
+              helpText="URN of SAML attribute containing the user's first name. Leave blank for default."
+            >
+              <label htmlFor="attrFirstName">First Name Attribute</label>
+              <Input
+                type="text"
+                id="attrFirstName"
+                name="attrFirstName"
+                defaultValue={config ? config.attrFirstName : ''}
+              />
+            </ValidationFormGroup>
+          </div>
+        </div>
+        <div className="row">
+          <div className="col col-4">
+            <ValidationFormGroup
+              for="attrLastName"
+              helpText="URN of SAML attribute containing the user's last name. Leave blank for default."
+            >
+              <label htmlFor="attrLastName">Last Name Attribute</label>
+              <Input
+                type="text"
+                id="attrLastName"
+                name="attrLastName"
+                defaultValue={config ? config.attrLastName : ''}
+              />
+            </ValidationFormGroup>
+          </div>
+        </div>
+        <div className="row">
+          <div className="col col-4">
+            <ValidationFormGroup
+              for="attrEmail"
+              helpText="URN of SAML attribute containing the user's email address[es]. Leave blank for default."
+            >
+              <label htmlFor="attrEmail">Email Address Attribute</label>
+              <Input
+                type="text"
+                id="attrEmail"
+                name="attrEmail"
+                defaultValue={config ? config.attrEmail : ''}
+              />
+            </ValidationFormGroup>
+          </div>
+        </div>
+
+        <div className="row">
+          <div className="col col-2">
+            <StatefulButton
+              state={submitState}
+              type="submit"
+              id="submitButton"
+              labels={{
+                default: 'Submit',
+                pending: 'Saving...',
+                complete: 'Complete',
+                error: 'Error',
+              }}
+              icons={{
+                default: <Icon className="fa fa-download" />,
+                pending: <Icon className="fa fa-spinner fa-spin" />,
+                complete: <Icon className="fa fa-check" />,
+                error: <Icon className="fa fa-times" />,
+              }}
+              disabledStates={[SUBMIT_STATES.PENDING]}
+              className="btn-primary ml-3 col"
+            />
+          </div>
+          <div className="col col-2">
+            {config && deleteEnabled &&
```

Comment:
Do we throw up a confirmation dialog before delete? perhaps we won't even have delete in the form anymore after some recent slack chats? or do we keep this?

Reply:
deleteEnabled is set to false, so this button is not visible. However, that is a good point about having a confirmation dialogue. Not sure if I should put it in or not. On the one hand, it could end up being a nontrivial amount of work, on the other it means if we get to the point of someone working on soft delete they theoretically shouldn't need to update the front end outside of setting deleteEnabled to true (or just removing that flag altogether really)

Reply:
yup let's hide for now and deal with the dialog if we bring it back yes